src/System.Private.ServiceModel/tools/setupfiles/sync-pr.cmd

```diff
@@ -0,0 +1,43 @@
+@echo off 
+setlocal
+
+REM 
+REM This calls a web service, specified on the command line, which syncs a 
+REM remote server's repo to the correct PR. 
+REM This script is meant to be called from a GitHub Pull Request. 
+REM If you must, you can fake it by setting the 'ghprbPullId' environment variable
+REM 
+
+set __args=%*
+
+echo.
+echo  WCF Remote Github Repo sync script
+
+if '%__args%'=='' (
+    echo   [%~n0] A URL must be specified for the pull request synchronization URL
+    echo     Usage:  %~n0 [sync url]
+    echo   sync-url - URL on remote server for PR synchronization
+    echo.
+    echo   Example:  %~n0 http://wcfci-sync-server/repo/sync.ashx
+    set __EXITCODE=1
+    goto done
+)
+
+set __SYNC_HOST_URL=%1
+
+if '%ghprbPullId%'=='' (
+    echo   [%~n0] This script should only be called only from the context of a GitHub Pull Request
```

Comment:
Not a deal breaker, but I still prefer parameters that optionally default to environment variables.  I know it is meant to be called by CI only, but the environment variable is a hidden contract, like using a global variable to pass method parameters.  I presume whatever logic calls this can simply pass the environment variable as the 2nd arg. Don't block progress on this -- it's just an attempt to reduce the "magic" that tends to surround CI scripts in general.